docs/repositories-configure/integrations/github-integration.md

```diff
@@ -48,3 +50,18 @@ Adds annotations on the lines of the pull request where Codacy finds new issues.
 Shows an overall view of the changes in the pull request, including new issues and metrics such as complexity and duplication. To enable this option, you must enable **Status checks** first.
 
 ![Pull request summary on GitHub](images/github-integration-pr-summary.png)
+
+### Suggest fixes
+
+Adds comments on the lines of the pull request where Codacy finds new issues with suggestions on how to fix the issues. Codacy doesn't apply any changes automatically. To apply the changes, manually review and accept the suggestions.
+
+<!-- TODO Add screenshot -->
+
+!!! note
+    This feature is in early access and has the following limitations:
+
+    -   The only supported Git providers are GitHub Cloud and GitHub Enterprise
+    -   The only tool that suggests fixes is ESLint for JavaScript
```

Comment:
Maybe we can remove the "for JacaScript part", as I think it also can suggest for Typescript, right @lolgab ?

Reply:
I actually forgot to ask about this. On our page detailing the supported languages and tools, ESLint is mentioned on JavaScript, JSON, and TypeScript. Does the tool provide suggestions for all these languages?

On the one hand, I would prefer not to mention the languages here since they may become out of date and it's an extra place to maintain them. On the other hand, if we do mention the languages here, it will help answer the question from the users: "Can I benefit from this feature or not?"

Reply:
Ops, ESLint doesn't run on JSON anymore. I'm going to open a PR to the documentation.
Both Javascript and Typescript are covered by suggestions. However, as you said, it's better not include them here to avoid future maintenance.

Reply:
Opened a PR here: https://github.com/codacy/docs/pull/445

Reply:
Not mentioning the languages makes even more sense because the idea is to include more supported tools. So the maintenance issue would only grow with time. :+1: